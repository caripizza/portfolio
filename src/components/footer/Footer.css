.Footer {
  ul {
    list-style: none;
    display: flex;
    flex-direction: column;
    padding: 2em 0 0 1.2em;
    margin: 0;
  }

  li {
    padding: 0;
    font-size: 0.8em;
  }

  a {
    display: inline-block;
  }
}
