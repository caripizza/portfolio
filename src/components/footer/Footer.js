import React from 'react';
import styles from './Footer.css';

const Footer = () => {
  const copyright = `Â© 2019-${new Date().getFullYear()} Cari Palazzolo, all rights reserved`;
  const linkedinLink = (
    <a href="https://linkedin.com/in/cari-palazzolo"
      target="_blank" rel="noopener noreferrer"
    >
      Linkedin
    </a>
  );
  const aboutLink = (
    <a href="https://caripalazzolo.com/about" target="_blank" rel="noopener noreferrer">About</a>
  );
  const github = (
    <a href="https://github.com/caripizza"
      target="_blank" rel="noopener noreferrer"
    >
      Github
    </a>
  );

  return (
    <footer className={styles.Footer}>
      <ul>
        <li>
          {copyright}
        </li>
        <li>
          {github} | {linkedinLink} | {aboutLink}
        </li>
      </ul>
    </footer>
  );
};

export default Footer;
