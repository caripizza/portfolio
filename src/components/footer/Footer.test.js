import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

describe('Footer', () => {
  it('matches a snapshot', () => {
    const component = renderer.create(
      <Footer />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
