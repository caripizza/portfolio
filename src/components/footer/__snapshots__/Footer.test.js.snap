// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches a snapshot 1`] = `
<footer
  className="Footer"
>
  <ul>
    <li>
      Â© 2019-2020 Cari Palazzolo, all rights reserved
    </li>
    <li>
      <a
        href="https://github.com/caripizza"
        rel="noopener noreferrer"
        target="_blank"
      >
        Github
      </a>
       | 
      <a
        href="https://linkedin.com/in/cari-palazzolo"
        rel="noopener noreferrer"
        target="_blank"
      >
        Linkedin
      </a>
       | 
      <a
        href="https://caripalazzolo.com/about"
        rel="noopener noreferrer"
        target="_blank"
      >
        About
      </a>
    </li>
  </ul>
</footer>
`;
